database/Host.js

```diff
@@ -0,0 +1,30 @@
+const mongoose = require('mongoose');
+const db = require('./index.js');
+mongoose.Promise = global.Promise;
+
+const hostSchema = new mongoose.Schema({
```

Comment:
I'm not really reviewing this schema in detail since I'm not as familiar with the data for your module, but it makes sense from what I can see. If you need something from my data that's not in my current schema, please let me know.